site/source/_posts/2019/01/28/Version-5-Dojo.md

```diff
@@ -15,18 +15,29 @@ We're excited to announce the 5.0.0 release of Dojo. This version builds on prev
 
 One of our primary goals for modern Dojo is to improve performance by serving minimal JavaScript bundles by default. For version 5, we're happy to announce an out of the box solution to optimize the user experience for dealing with bundling and loading polyfills in Dojo applications.
 
-By using `@dojo/framework/shim` ponyfills for [`WebAnimations`][WebAnimationsAPI], [`IntersectionObserver`][IntersectionObserverAPI], [`ResizeObserver`][ResizeObserverAPI], and [`fetch`][FetchAPI] the Dojo build produces separate "platform" bundles that will be **only** be loaded based on two key conditions:
+The Dojo build produces separate "platform" bundles that will be **only** be loaded based on two key conditions:
 
 1) The shim module is imported somewhere in an application.
 1) A users browser does not natively support the browser feature.
 
 In short, if your application does not use the browser feature or you use a browser with native support, the polyfill will not be loaded! This change means serving less JavaScript and improving your application performance without compromising on features.
 
+### New Polyfills Added to Shim
+
+We've added additional third party polyfills to `@dojo/framework/shim`:
+
+* [`fetch`][FetchAPI] - The Fetch API provides an interface for fetching resources (including across the network).
```

Comment:
nice! might be worth adding the import here instead of just `fetch` etc? ie `@dojo/framework/shim/fetch`?

Reply:
Nice idea